addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -229,8 +234,15 @@ class DescriptionScreen extends Component {
      * @param {Object} ui an object with label and value properties for
      *      the selected option.
      */
-    _selectIndustry(ev, ui) {
+    async _selectIndustry(ev, ui) {
         if (this.autocompleteHasResults) {
+            if (this.sendIap.length) {
```

Comment:
```suggestion
            if (this.sendIap) {
```
Can't have empty term, right?